### PR TITLE
Press release prep

### PR DESCRIPTION
**Pull request by LEQUI000**

### CODE REVIEW

**Comment by LEQUI000** [comment]
@CurtisGreen I don't think nested bullet points are currently working can you take a look?

**Comment by LEQUI000** [comment]
Need to replace with a smaller version of the logo

**Comment by LEQUI000** [comment]
PLEASE DELETE PHONE NUMBER IF WE CHOOSE NOT TO UTILIZE THE PHONE OPTION

File: _posts/data-announces-formation.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ Tyler Wright, Interim Co-Chair
 Connor Hulla, Interim Co-Chair  
 Dallas Area Transit Alliance (DATA)  
 Email: [info@ridewithdata.org](mailto:info@ridewithdata.org)   
-Website: [dallasareatransitalliance.org](https://dallasareatransitalliance.org)  
+Website: [dallasareatransitalliance.org](https://dallasareatransitalliance.org)  and [ridewithdata.org](https://ridewithdata.org)
```
Comment (CurtisGreen) [inline _posts/data-announces-formation.md]:
```suggestion
Website: [dallasareatransitalliance.org](https://dallasareatransitalliance.org) or [ridewithdata.org](https://ridewithdata.org)  
```

Comment (LEQUI000) [reply _posts/data-announces-formation.md]: Fixed in commit https://github.com/CurtisGreen/dallas-area-transit-alliance/pull/13/commits/acce2d4e67fd760fae86959d5709ceb6f9517da3

**Comment by LEQUI000** [comment]
I still need to edit the social media handles once confirmed by Connor plus add the Facebook links to the groups and pages.

Also add the banner property so a banner will show up once published.

**Comment by LEQUI000** [comment]
Updated, will change to only add a post once this commit is live to announce the formation.

**Review by Hexel — APPROVED** [review]

**Pull request merged**